lua/tabline/utils.lua

```diff
@@ -34,4 +34,67 @@ M.get_tabname = function(bufname, index)
     return vim.fn.fnamemodify(bufname, ':t')
 end
 
+M.is_val_in_table = function(val, tbl)
```

Comment:
I think you can use: `vim.tbl_contains` instead of this?

Or in the case I see that it's used, I think `vim.list_contains` is a better fit

Reply:
Ah, good. Wasn't aware. Will change.